### PR TITLE
Support for WyrmRoost mod

### PR DESCRIPTION
**Pull request by najlitarvan**
added the wyrmroost, had trouble putting in json so it is txt but is was written while it was json. heavy use of comment for easier tweaking in the future

### CODE REVIEW

File: inputs/wyrmroost.txt [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            "metal":4,
+                        // it's a living being in an inventory (dragon with a golden nugget)
+        },
+        "blue_geode":{
+            "crystal":200,
+            "weird":15,
+            //cheaper diamond like item with some magical property, overworld
+        },
+        "blue_geode_ore":{
+            "crystal":200,
```
Comment (Entoarox) [inline inputs/wyrmroost.txt]: Ores need to have the goo value `"logic_forbidden":1` on them, this is what tells Goo that the item cannot be solidified.

File: inputs/wyrmroost.txt [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            //cheaper diamond like item with some magical property, overworld
+        },
+        "red_geode":{
+            "crystal":200,
+            "weird":15,
+            "decay":5,
+            "molten":5,
+            // stronger version of the geode before it, found in nether
+        },
+         },
```
Comment (Entoarox) [inline inputs/wyrmroost.txt]: JSON formatting error, you have `},` repeated when it should not be.

File: inputs/wyrmroost.txt [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        "red_geode":{
+            "crystal":200,
```
Comment (Entoarox) [inline inputs/wyrmroost.txt]: You repeated the same item here, duplicating entries has no effect, only the last one defined will actually be used.

File: inputs/wyrmroost.txt [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+            "weird":15,
+            "decay":5,
+            "molten":5,
+            // stronger version of the geode before it, found in nether
+        },
+        "red_geode_ore":{
+            "crystal":200,
```
Comment (Entoarox) [inline inputs/wyrmroost.txt]: Same remark about `logic_forbidden` here again.

File: inputs/wyrmroost.txt [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+            // stronger version of the geode before it, found in nether
+        },
+        "purple_geode":{
+            "crystal":200,
+            "weird":20,
+            "vital":1,
+            "earthen":120,
+            // end geode
+        },
+        "purple_geode_ore":{
```
Comment (Entoarox) [inline inputs/wyrmroost.txt]: Another `logic_forbidden` situation.

File: inputs/wyrmroost.txt [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+            "crystal":200,
+            "weird":20,
+            "vital":1,
+            "earthen":120,
+            // end geode
+        },
+        "platinum_ingot":{
```
Comment (Entoarox) [inline inputs/wyrmroost.txt]: Platinum is a common material, it should setup to inherit from a common prototype rather then being explicitly defined here.

File: inputs/wyrmroost.txt [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+            "regal":80,
+            "metal":54,
+            
+            // stronger gold
+        },
+        "platinum_ore":{
```
Comment (Entoarox) [inline inputs/wyrmroost.txt]: Same with platinum ore.

**Review by Entoarox — CHANGES_REQUESTED** [review]
I would also recommend using some mod-specific protoypes (See quark.json) for the geodes, since they all share the same Goo value.

File: inputs/wyrmroost.txt (hunk 7 above)
Comment (najlitarvan) [reply inputs/wyrmroost.txt]: where can i then find the list of already defined prototypes i saw biomes of plenty but no quark or other mods

File: inputs/wyrmroost.json [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+            "metal":4,
+                        // it's a living being in an inventory (dragon with a golden nugget)
+        },
+        "blue_geode":{
+            "crystal":200,
+            "weird":15,
+            //cheaper diamond like item with some magical property, overworld
+        },
+        "blue_geode_ore":{
+	"logic_forbidden":1,
```
Comment (Entoarox) [inline inputs/wyrmroost.json]: Indentation on this needs to match that on the `"crystal":200,` on the line below.

File: inputs/wyrmroost.json [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+        "red_geode":{
+            "crystal":200,
+            "weird":15,
+            "decay":5,
+            "molten":5,
+            // stronger version of the geode before it, found in nether
+        },
+         
+        "red_geode_ore":{
+	"logic_forbidden":1,
```
Comment (Entoarox) [inline inputs/wyrmroost.json]: Same issue with indentation, keeping it clean makes it easier to read this stuff.

File: inputs/wyrmroost.json [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+        },
+        "purple_geode":{
+            "crystal":200,
+            "weird":20,
+            "vital":1,
+            "earthen":120,
+            // end geode
+        },
+        "purple_geode_ore":{
+	"logic_forbidden":1,
```
Comment (Entoarox) [inline inputs/wyrmroost.json]: Indentation again.

File: inputs/wyrmroost.json [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+        },
+        "cooked_behemoth_meat":{
+            "vital":15,
+            "faunal":100,
+            
+            // different tiers of meat higher the tier>lower in this menu and way higher values when cooked T4 H 10 S 40 :-) not joking
+		},
+            // added by prototype
+		"platinum_ingot":"platinum",
+		"platinum_ore":"platinum_ore",
```
Comment (Entoarox) [inline inputs/wyrmroost.json]: This whole section is indented a bit more then it should be.

**Review by Entoarox — CHANGES_REQUESTED** [review]

**Review by Entoarox — APPROVED** [review]

**Pull request merged**